### DISCUSSION TITLE
Link grouping by tags or categories or hashtags

### DISCUSSION THREAD

**Discussion by yeldarx** [YOURLS/YOURLS#3123] [Plugins]
**Summary:** It seems we need to add the ability to group links. There are several options: keywords, categories, tags, hashtags, etc.

**Motivation:** Over time, there are a lot of links and sometimes it is difficult to find the one you need. Let's say a person writes a book and shortens all the links for his book through YOURLS - he needs to group all the links from his book in order to correct the desired link if necessary.

**Solution:** It should be a plugin that allows you to add to the shortened link keyword or category or tag or hashtags, etc.

**Comment by DigitalPeace** [comment]
I was searching for a plugin that would group link by categories and found your post.

It would be great to see this type of functionality implemented.
